### DISCUSSION TITLE
LightningCLI to Sagemaker Script

### DISCUSSION THREAD

**Discussion by Zasder3** [Lightning-AI/pytorch-lightning#17887] [DDP / multi-GPU / multi-node]
I have a training regime that works through the lightning CLI. Now, I'm trying to port it over Sagemaker to do multi-GPU training with DDP. What is the workflow for porting a Trainer that uses LightningCLI over to Sagemaker? Are there any examples or any major landmarks that need to be accomplished?

I've tried searching for anything that does this, but have come up empty-handed.

**Comment by celsofranssa** [comment]
Any update on this? Is there anything similar to [W&B Launch](https://docs.wandb.ai/guides/launch/sagemaker)?

**Reply by mauvilsa** [reply]
There is a post about using LightningCLI in SageMaker. I have not used it or know the details. Just linking it here in case it is useful.
- https://krasserm.github.io/2022/01/21/sagemaker-multi-node/
- https://krasserm.github.io/2022/02/26/sagemaker-fault-tolerance/

**Reply by celsofranssa** [reply]
Hello @mauvilsa, thank you.
